dll/cpl/appwiz/createlink.c

```diff
@@ -376,6 +388,15 @@ FinishDlgProc(HWND hwndDlg,
                     }
                 }
             }
+            else if (lppsn->hdr.code == PSN_WIZBACK)
+            {
+                PropSheet_SetWizButtons(GetParent(hwndDlg), PSWIZB_NEXT);
```

Comment:
Setting the wizard buttons for the current page must be done by that page for consistency purposes.